website/docs/getting-started/setup/instance-configuration/README.mdx

```diff
@@ -0,0 +1,146 @@
+---
+description: >-
+  Appsmith ships with third-party services & configurations that improve the app
+  building experience. All services & configurations are entirely optional.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Instance Configuration
+
+Configuration of your Appsmith instance allows you to tailor it to your specific needs. This process includes setting up elements like environment variables, authentication, emails, SSL, and custom domain. Proper configuration ensures that your instance is running efficiently. 
+
+## Configure Docker installations
+
+To configure a docker installation, simply open the folder of your installation and edit the **`stacks/configuration/docker.env`** file with the environment variables for the service.
+
+Remove # before the variables to ensure they're not commented
+
+```bash
+// Example variable configuration format
+APPSMITH_GOOGLE_MAPS_API_KEY=YOUR_API_KEY
+```
+
+After making any changes, **remember to restart the docker containers** for the changes to take effect.
+
+```bash
+// To restart Appsmith using docker
+docker restart appsmith
+
+// To restart Appsmith using docker compose
+docker-compose restart appsmith
+```
```

Comment:
Similar problem as above. Let's remove comments, _and_ split this into two code blocks, with the comment as just ordinary text, so it's easy to copy just any one command.